tests/test_harvesting.py

```diff
@@ -0,0 +1,65 @@
+# along with Invenio; if not, write to the Free Software Foundation, Inc.,
+# 59 Temple Place, Suite 330, Boston, MA 02111-1307, USA.
+from invenio.testsuite import InvenioTestCase, make_test_suite, run_test_suite
+from invenio_oaiharvester.manage import get
+
+
+class OaiHarvesterTests(InvenioTestCase):
+
+    def test_get_from_identifiers(self):
+        test_record = {
+            'license': ['http://arxiv.org/licenses/nonexclusive-distrib/1.0/'],
+            'forenames': ['J. E.', 'Rachel Kuzio', 'Sharon X.'],
+            'abstract': ['  We introduce the MUSCEL Program (MUltiwavelength observations of the\n'
+                         'Structure, Chemistry and Evolution of LSB galaxies), a project aimed at\n'
+                         'determining the star-formation histories of low surface brightness galaxies.\n'
+                         'MUSCEL utilizes ground-based optical spectra and space-based UV and IR\n'
+                         'photometry to fully constrain the star-formation histories of our targets with\n'
+                         'the aim of shedding light on the processes that led low surface brightness\n'
+                         'galaxies down a different evolutionary path from that followed by high surface\n'
+                         'brightness galaxies, such as our Milky Way. Here we present the\n'
+                         'spatially-resolved optical spectra of UGC 628, observed with the VIRUS-P IFU at\n'
+                         'the 2.7-m Harlen J. Smith Telescope at the McDonald Observatory, and utilize\n'
+                         'emission-line diagnostics to determine the rate and distribution of star\n'
+                         'formation as well as the gas-phase metallicity and metallicity gradient. We\n'
+                         'find highly clustered star formation throughout UGC 628, excluding the core\n'
+                         'regions, and a log(O/H) metallicity around -4.2, with more metal rich regions\n'
+                         'near the edges of the galactic disk. Based on the emission-line diagnostics\n'
+                         'alone, the current mode of star formation, slow and concentrated in the outer\n'
+                         'disk, appears to have dominated for quite some time, although there are clear\n'
+                         'signs of a much older stellar population formed in a more standard inside-out\n'
+                         'fashion.\n'],
+            'author': [None, None, None],
+            'comments': ['13 pages, 7 figures'],
+            'created': ['2015-07-10'],
+            'keyname': ['Young', 'de Naray', 'Wang'],
+            'authors': [None],
+            'title': ['The Distribution of Star Formation and Metals in the Low Surface\n  Brightness Galaxy UGC 628'],
+            'id': ['1507.03011'],
+            'categories': ['astro-ph.GA']}
+
+        records = get('arXiv', 'arxiv_math_daily', None, 'oai:arXiv.org:1507.03011', None, None, None)
```

Comment:
To avoid this test being bound to a database object (or even a remote site), you can take the following steps:

- [ ] Change the parameters for now to use `url` instead of `name` (to not require database)
- [ ] Take advantage of the [httpretty](https://github.com/gabrielfalcao/HTTPretty#in-a-nutshell) library to mock remote requests